src/lib.rs

```diff
@@ -66,7 +66,7 @@ mod request;
 mod response;
 
 pub use append_only_data::{
-    Address as ADataAddress, AppendOnlyData, Index as ADataIndex, Indices as ADataIndices,
+    Action, Address as ADataAddress, AppendOnlyData, Index as ADataIndex, Indices as ADataIndices,
```

Comment:
I think this should be
```suggestion
    Action as ADataAction, Address as ADataAddress, AppendOnlyData, Index as ADataIndex, Indices as ADataIndices,
```
to be not confused with `MDataAction`.